### PR TITLE
separated files for Active and Complete

### PR DESCRIPTION
**Pull request by aman7870**
Please review made files for Active and Complete task.

### CODE REVIEW

**Review by RohitKuwar — DISMISSED** [review]

File: src/pages/completed.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    error,
+    isLoading,
+  } = useFetch(TASKS_URL);
+
+  useEffect(() => {
+    if ('tasks' in response) {
+      setTasks(response.tasks);
+
+      const complete = tasks.filter(
+        (item: task) => item.status === 'Completed',
```
Comment (Rucha1499) [inline src/pages/completed.tsx:32]: `Completed` in lower case?

File: src/pages/active.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    response,
+    error,
+    isLoading,
+  } = useFetch(TASKS_URL);
+
+  useEffect(() => {
+    if ('tasks' in response) {
+      setTasks(response.tasks);
+      const active = tasks.filter(
+        (item: task) => item.status === 'Active',
```
Comment (Rucha1499) [inline src/pages/active.tsx:30]: We are receiving this from the backend right? Shouldn't the value of status be in lowercase? 
image

**Review by Rucha1499 — COMMENTED** [review]
Nice work! Kindly have a look at review comment. 😃

File: src/pages/completed.tsx (hunk 1 above)
Comment (Rucha1499) [reply src/pages/completed.tsx:32]: please refer https://github.com/Real-Dev-Squad/website-status/pull/97#discussion_r604332513

**Review by whyDontI — CHANGES_REQUESTED** [review]
- We do not need the accordion to display just one type of task, it'll be great if you can remove the accordion.
- Please add a message when there are no tasks available, if it is there please make sure it's working correctly
Screenshot from 2021-03-31 01-10-35

**Comment by aman7870** [comment]
Hi @whyDontI, I did change according to the requested. I have attached screenshots for changes do have a look and please 
active
complete

review.

**Review by whyDontI — APPROVED** [review]
Good work @aman7870 🥳 
Thanks for the PR

**Pull request merged**